web/client/translations/data.de-DE

```diff
@@ -1868,6 +1868,8 @@
             "addTextContent": "Textinhalt hinzufügen",
             "addMediaContent": "Medieninhalt hinzufügen",
             "zoomToContent": "Zum Inhalt zoomen",
+            "emptyTitle": "This story is empty",
+            "emptyDescription": "Start creating an awesome geostory by adding new content",
```

Comment:
mmm...I know I suggested to avoid localizing until text was confirmed, but if you have 2 strings and you put them in tranaslations, maybe is a good idea to do it now. 
Please localize de, it, es, fr

Reply:
yeah, it was to ease translation after, but ok